c_binding/include/libstoragemgmt/libstoragemgmt_local_disk.h

```diff
@@ -26,6 +26,25 @@
 extern "C" {
 #endif
 
+#define LSM_LOCAL_DISK_RPM_UNKNOWN                  -1
+#define LSM_LOCAL_DISK_RPM_NON_ROTATING_MEDIUM      0
+
+typedef enum {
+    LSM_LOCAL_DISK_LINK_TYPE_UNKNOWN =  -1,
+    LSM_LOCAL_DISK_LINK_TYPE_FC = 0,
+    LSM_LOCAL_DISK_LINK_TYPE_SSA = 2,
+    LSM_LOCAL_DISK_LINK_TYPE_SBP = 3,
+    LSM_LOCAL_DISK_LINK_TYPE_SRP = 4,
+    LSM_LOCAL_DISK_LINK_TYPE_ISCSI = 5,
+    LSM_LOCAL_DISK_LINK_TYPE_SAS = 6,
+    LSM_LOCAL_DISK_LINK_TYPE_ADT = 7,
+    LSM_LOCAL_DISK_LINK_TYPE_ATA = 8,
+    LSM_LOCAL_DISK_LINK_TYPE_USB = 9,
```

Comment:
I am using value from T10 SPC-5 directly(which is hex), just in case of human convention error.